core/types/vote.go

```diff
@@ -0,0 +1,89 @@
+package types
+
+import (
+	"sync/atomic"
+
+	"github.com/ethereum/go-ethereum/crypto/bls"
+	"github.com/pkg/errors"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	BLSPublicKeyLength = 48
+	BLSSignatureLength = 96
+
+	MaxAttestationExtraLength = 256
+)
+
+type BLSPublicKey [BLSPublicKeyLength]byte
+type BLSSignature [BLSSignatureLength]byte
```

Comment:
These types are in crypto/bls package already, we shouldn't redefine them here

Reply:
these are purely slices not the object itself

Reply:
But why don't we use common.PublicKey and common.Signature from crypto/bls/common/interface.go?

Reply:
we also store pubKey and secret key at slice of bytes in keymanager. Therefore we keep these type at byte also